recipes/daphne/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "daphne" %}
+{% set version = "2.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 1e41d02d78da373c1f8504835cdca6d447b5d64c2941661d4bd968d412b6c383
+
+build:
+  number: 0
+  entry_points:
+    - daphne = daphne.cli:CommandLineInterface.entrypoint
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - asgiref >=3.2,<4
+    - autobahn >=0.18
+    - pip
+    - python
+    - twisted[tls] >=18.7
+    - pytest-runner
+  run:
+    - asgiref >=3.2,<4
+    - autobahn >=0.18
+    - python
+    - twisted[tls] >=18.7
+    - pytest-runner
```

Comment:
I would think you don't need `pytest-runner` to run the package. I think it's only required in the `setup.py`